docker/docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+version: '3.8'
+
+# Custom top-level network
+networks:
+  internal:
+  db:
+  web:
+
+services:
+  neo4j:
+    image: ${NEO4J_DOCKER_IMAGE:-neo4j:4.4}
```

Comment:
How is this one populated? Does this env variable need to be added to the `.env.template` and `.env.testing` files?

Reply:
Ah, I missed that! The syntax here: 
`${NEO4J_DOCKER_IMAGE:-neo4j:4.4}` 
means "If `NEO4J_DOCKER_IMAGE` is unset, use `neo4j:4.4`.
Yay crazy bash syntax.

Reply:
Ah cool, that makes more sense!